runtime/interpreter/function.go

```diff
@@ -101,6 +102,12 @@ func (f *InterpretedFunctionValue) RecursiveString(_ SeenReferences) string {
 	return f.String()
 }
 
+func (f *InterpretedFunctionValue) ToMeteredString(memoryGauge common.MemoryGauge, _ SeenReferences) string {
+	typeString := f.Type.String()
```

Comment:
We need to separately meter string conversions of runtime types. I'll open a subsequent PR for that